test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Build.AspNetCore3x/Microsoft.Test.E2E.AspNetCore3x.OData.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
     <PackageReference Include="Microsoft.OData.Client" Version="7.6.1" />
+    <PackageReference Include="System.Diagnostics.EventLog" Version="4.6.0" />
```

Comment:
This fixes an issue being reported on ADO even for other PRs:
```
Error CS1069: The type name 'EventLog' could not be found in the namespace 'System.Diagnostics'
```
More details https://stackoverflow.com/questions/57688012/reading-from-eventlog-error-the-type-name-eventlog-could-not-be-found-in-the